src/bootstrap/native.rs

```diff
@@ -568,7 +568,7 @@ fn configure_cmake(
     // We also do this if the user explicitly requested static libstdc++.
     if builder.config.llvm_static_stdcpp {
         if !target.contains("msvc") && !target.contains("netbsd") {
-            if target.contains("apple") {
```

Comment:
This looks like unintended change

Reply:
This is a drive-by fix for https://github.com/rust-lang/rust/pull/94832#issuecomment-1073971120 without which Rust doesn't build on my machine.